clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -78,6 +82,21 @@ type Controller struct {
 	ssh util.SSHExecutor
 }
 
+// CompleteConfig completes Prometheus manifest file path config
+func CompleteConfig(p *config.PrometheusConfig) {
+	if p.ManifestPath != "" {
+		coreManifests = p.ManifestPath + "/*.yaml"
+		defaultServiceMonitors = p.ManifestPath + "/default/*.yaml"
+		masterIPServiceMonitors = p.ManifestPath + "/master-ip/*.yaml"
+		nodeExporterPod = p.ManifestPath + "/exporter/node-exporter.yaml"
+	} else {
+		coreManifests = "$GOPATH/src/k8s.io/perf-tests/clusterloader2/pkg/prometheus/manifests/*.yaml"
```

Comment:
Please keep the prefix `$GOPATH/src/k8s.io/perf-tests/clusterloader2/pkg/prometheus/manifests` as const.  With this the whole code can be simplified to:

```
prefix := p.ManifestPath
if p.ManifestPath == "" {
  prefix = defaultPrometheusManifestPath
}
coreManifests = prefix + "/*.yaml"
defaultServiceMonitors = prefix + "/default/*.yaml"
masterIPServiceMonitors = prefix + "/master-ip/*.yaml"
nodeExporterPod = prefix + "/exporter/node-exporter.yaml"
```

Reply:
+1